src/styles/nextButton.scss

```diff
@@ -1,23 +1,30 @@
 @import 'global';
 
-.nextButton {
+#next-button-container {
   background-color: $main-orange;
-  border: 0;
   border-radius: 20px;
-  bottom: 30px;
-  color: black;
+  bottom: 20px;
   cursor: default;
-  font-family: $main-font;
-  font-size: 24px;
-  height: 37px;
-  line-height: 23px;
   opacity: 50%;
-  padding: 5px 20px;
+  padding: 10px 20px;
   position: absolute;
-  right: 30px;
-  text-align: center;
+  right: 20px;
+}
+
+@media only screen and (max-width: 1000px) {
+  #next-button-container {
+    right: calc(50% - 45px);
+  }
+}
+
+.nextButton {
```

Comment:
Having a container for the next-button causes issues when enabled. Can we try to keep it as one button class and do something like 
```suggestion
.nextButton {
  background-color: $main-orange;
  border: 0;
  border-radius: 20px;
  bottom: 20px;
  color: black;
  cursor: default;
  font-family: $main-font;
  font-size: 24px;
  line-height: 10px;
  opacity: 50%;
  padding-bottom: 15px;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  position: absolute;
  right: 20px;
  text-decoration: none;
}
```
 some values may have to be adjusted and nextButton.tsx will have to be changed too. Lmk if that makes sense!

Reply:
fixed now